eng/scripts/Install-GitHooks.ps1

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env pwsh
+<#
+.SYNOPSIS
+    Installs git hooks from eng/scripts/git-hooks to .git/hooks
+.DESCRIPTION
+    This script copies git hook scripts from eng/scripts/git-hooks to .git/hooks
+    and ensures they are executable on all platforms.
+#>
+
+[CmdletBinding()]
+param()
+
+$repoRoot = Resolve-Path (Join-Path $PSScriptRoot "../..")
+$gitHooksSourceDir = Join-Path $repoRoot "eng/scripts/git-hooks"
+$gitHooksTargetDir = Join-Path $repoRoot ".git/hooks"
+
+# Ensure the .git/hooks directory exists
+if (-not (Test-Path $gitHooksTargetDir)) {
+    Write-Host "Creating $gitHooksTargetDir directory..."
+    New-Item -Path $gitHooksTargetDir -ItemType Directory -Force | Out-Null
+}
+
+# Remove existing pre-commit hook
+Remove-Item -Path $gitHooksTargetDir/pre-commit -Force -ErrorAction SilentlyContinue | Out-Null
+
+# Copy hook files
+Write-Host "Copying git hooks from $gitHooksSourceDir to $gitHooksTargetDir..."
+$hookFiles = Get-ChildItem -Path $gitHooksSourceDir -File
+
+foreach ($file in $hookFiles) {
+    $targetFile = Join-Path $gitHooksTargetDir $file.Name
+
+    Write-Host "  Copying $($file.Name)..."
+    Copy-Item -Path $file.FullName -Destination $targetFile -Force
```

Comment:
Instead of the hard coded delete of pre-commit earlier maybe do a rename of x to x.old right before this line.

Reply:
I'll need both.   We're not adding a new pre-commit hook, but we are wanting to remove the existing one

Reply:
I added the back to .old as well as special casing the pre-commit hook delete if it matches pre-push